api/templates/design.go

```diff
@@ -41,6 +42,9 @@ var index = ResultType("application/vnd.templates.index", func() {
 
 var _ = Service("templates", func() {
 	Description("Service for applying templates to Keboola projects")
+	cors.Origin("*", func() {
+		cors.Headers("X-KBC-RunId", "X-StorageApi-Token")
```

Comment:
Budeme na nieco pouzivat `X-KBC-RunId`?

Reply:
Nevím, zatím asi ne, dal jsem to tam pro jistotu. 😄 V jiných službách v PS se to používá. Vlastně do logu do Papertrailu se to bude hodit. Pak pro případný debugování nějakých requestů napříč apičkama.

Reply:
No neviem ci to dava zmysel, ale dobre :-) moze byt ...
`RunID` je IDe jobu, nie? 
Zatial neviem o tom, ze by sa API malo volat z nejakeho jobu.

Reply:
Jo jasně, tak až to api začne někdo volat z nějaký komponenty. 😅 Ne dobře, oddělám to zatím. Přidat se to dá vždycky.